tests/integrate_test/test_perm_denied.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+"""
+Test case of checkCreateContractPermission in sysconfig
+"""
+import functools
+import subprocess
+import time
+from jsonrpcclient.http_client import HTTPClient
+
+LATEST_VERSION = 1
+
+
+def send_tx(privkey, code="", version=LATEST_VERSION):
```

Comment:
Would we refactor the `send_tx`, `get_receipt` and `rpc request` first?
Check the `tests/integrate_test/*.py`

Reply:
Yeal, There should be a `utils.py`, contains these functions.

Reorganize the whole integrate test directory firstly.